.circleci/config.yml

```diff
@@ -5,116 +5,125 @@ jobs:
 
   # DEPLOY-STAGING JOB
   # installing dependencies, building assets and deploying to staging
-  deploy-staging:
+  build-and-test-uikit:
     docker:
-      - image: circleci/node:8.9.1
-
+      - image: circleci/node:8.11.3                       # NodeLTS as of 07/2018
```

Comment:
fixing it to major only will circumvent issues with new releases... just do `image: circleci/node:8`

Reply:
@dominikwilkowski I respectfully disagree, targeting the latest `LTS` version should be recommended as it's the most stable, secure and tested. It also isolates the project build from potential upstream issues.

This is also specified as [best practice from CircleCI:](https://circleci.com/docs/2.0/circleci-images/#best-practices)

> Convenience images are based on the most recently built versions of upstream images, so it is best practice to use the most specific image possible. This makes your builds more deterministic by preventing an upstream image from introducing unintended changes to your image.

Reply:
Yeah fair but this feels wrong. You now have to intervene when there is a urgent patch. I can't see anything wrong with pinning to major version.

Reply:
Thanks @dominikwilkowski I think you make a great point. For now I think we will leave it as is `8.11.3` ( we did this before as well ). 

I have had issues in the past with cloud foundry containers changing how they use `node_modules` and it breaking previously working builds. It makes it quite hard to troubleshoot when the image changes something like this.